README.md

```diff
@@ -197,28 +197,28 @@ mode: single
 
 ### Octopus Energy Plugin
 - If you want to use real pricing data and have Octopus Energy then ensure you have the Octopus Energy plugin installed and working (https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/)
+- The Octopus Energy Plugin also provides support for Intelligent Octopus charging to support car charging (although you can also use the Octopus Intelligent Plugin if you prefer, see below)
```

Comment:
The changes look good, although I did wonder if removing references to the Intelligent Plugin might make it read better, I don't see the point in using that one anymore?

Reply:
Yep, I'm happy to do that, I'll go through and sort that

Reply:
I've merged these now for, it's probably easy to tidy up afterwards. Thanks for your help.